pkg/apis/provisioning/v1alpha5/labels.go

```diff
@@ -38,6 +38,7 @@ var (
 	TerminationFinalizer              = Group + "/termination"
 	LabelNodeInitialized              = Group + "/initialized"
 	LabelCapacityType                 = Group + "/capacity-type"
+	DriftedAnnotationKey              =	Group + "/drifted-node"
```

Comment:
I'd love to abstract this concept into `karpenter.sh/voluntary-disruption = drifted`. @jonathan-innis can you chime in? This enables us to write additional integrations for other case like node rebalance, etc. 

```
karpenter.sh/voluntary-disruption = drifted
karpenter.sh/voluntary-disruption = rebalance
karpenter.sh/voluntary-disruption = expired
```

Reply:
Happy to do this way, I think @njtran in https://github.com/aws/karpenter/pull/2812 mentioned to use this as a label. Let me know and I would convert this to a label as you mentioned.

Reply:
He and I are on the same page. Happy to accept as you have it, and we'll tidy it up later. @jonathan-innis FYI.